Formula/pdmenu.rb

```diff
@@ -0,0 +1,34 @@
+class Pdmenu < Formula
+  desc "Full screen menuing system for Unix"
+  homepage "https://joeyh.name/code/pdmenu/"
+  url "http://sources.buildroot.net/pdmenu/pdmenu-1.3.6.tar.gz"
+  sha256 "dedd2a4a5719278b5e49041161990c2f20b5909818837542aaca01819f2c14eb"
+  license "GPL-2.0-only"
+
+  head "git://git.joeyh.name/pdmenu"
+
+  depends_on "gettext" => :build
+  depends_on "s-lang" => :build
+
+  def install
+    ENV.append "CC", "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append "SLANG_H_LOC", "#{HOMEBREW_PREFIX}/include/slang.h"
+    ENV.append "SLANG_LIB_LOC", "#{HOMEBREW_PREFIX}/lib"
```

Comment:
```suggestion
    ENV["SLANG_H_LOC"] = Formula["s-lang].opt_include/"slang.h"
    ENV["SLANG_LIB_LOC"] = Formula["s-lang"].opt_lib
```
Have you tried building without these variables? I suspect that they're not needed.

Reply:
Of course, the configure script does not find slang.h nor libslang.*

Reply:
same response: needed manually, not needed in the formula...